rosdep/python.yaml

```diff
@@ -1513,6 +1513,22 @@ python-fastdtw-pip:
   ubuntu:
     pip:
       packages: [python-fastdtw]
+python3-fastnumbers-pip:
```

Comment:
This is failing CI because the whole section has to move down into the `python3-` section.

Reply:
Oooooh, I was wondering why it failed. Sorry, I overread the section on `python-` vs `python3-` placements.